app/wama.web/Controllers/CheckInController.cs

```diff
@@ -29,10 +31,11 @@ public IActionResult Index()
         [HttpPost]
         public async Task<IActionResult> Index(string memberId)
         {
-            if (ModelState.IsValid && !string.IsNullOrWhiteSpace(memberId))
+            if (ModelState.IsValid &&  !string.IsNullOrWhiteSpace(memberId) && (memberId.Length == 6 || memberId.Length == 7))
```

Comment:
Nice, but not quite. I think there's a better way to achieve this with attributes